ui/components/app/cancel-speedup-popover/cancel-speedup-popover.js

```diff
@@ -98,54 +97,51 @@ const CancelSpeedupPopover = () => {
       <div className="cancel-speedup-popover__wrapper">
         <Text
           alignItems={AlignItems.center}
-          display={DISPLAY.FLEX}
+          display={Display.Flex}
           variant={TextVariant.bodySm}
-          as="h6"
-          marginTop={0}
           marginBottom={2}
+          paddingBottom={2}
+          className="cancel-speedup-popover__description"
         >
           {t('cancelSpeedUpLabel', [
-            <strong key="cancelSpeedupReplace">{t('replace')}</strong>,
+            <strong key="cancelSpeedupCancel">{t('replace')}</strong>,
           ])}
           <InfoTooltip
             position="top"
             contentText={
-              <Box>
-                {t('cancelSpeedUpTransactionTooltip', [
-                  editGasMode === EditGasModes.cancel
-                    ? t('cancel')
-                    : t('speedUp'),
-                ])}
-                <div>
-                  <a
-                    href="https://community.metamask.io/t/how-to-speed-up-or-cancel-transactions-on-metamask/3296"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                  >
-                    {t('learnMoreUpperCase')}
-                  </a>
-                </div>
-              </Box>
+              <>
+                <Text variant={TextVariant.bodySm}>
+                  {t('cancelSpeedUpTransactionTooltip', [
+                    editGasMode === EditGasModes.cancel
+                      ? t('cancel')
+                      : t('speedUp'),
+                  ])}
+                </Text>
+                <ButtonLink
+                  variant={TextVariant.bodySm}
+                  href="https://community.metamask.io/t/how-to-speed-up-or-cancel-transactions-on-metamask/3296"
+                  target="_blank"
+                >
+                  {t('learnMoreUpperCase')}
+                </ButtonLink>
+              </>
```

Comment:
Is the choice to remove the <Box> in favor of a fragment related to the issue?

Reply:
Probably not particularly related to the issue but could have some minor performance benefits as there are no utility props being used on the `Box` so could save a small amount of render time?

Reply:
Cool